build/resources/main/web/style.css

```
@@ -0,0 +1,699 @@
+}
+.item-icon img {
+    width: 32px;
+    height: 32px;
+    image-rendering: pixelated;
+}
+.item-name {
+    font-size: 14px;
+    font-weight: 600;
+    color: var(--text-primary);
+    line-height: 1.3;
+    word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)